e2etest/zt_newe2e_workload_test.go

```diff
@@ -0,0 +1,70 @@
+package e2etest
+
+import (
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"os"
+	"time"
+)
+
+func init() {
+	suiteManager.RegisterSuite(&WorkloadIdentitySuite{})
+}
+
+type WorkloadIdentitySuite struct{}
+
+// Run only in environments that support and are set up for Workload Identity (ex: Azure Pipeline, Azure Kubernetes Service)
+func (s *WorkloadIdentitySuite) Scenario_SingleFileUploadDownloadWorkloadIdentity(svm *ScenarioVariationManager) {
+	// Run only in environments that support and are set up for Workload Identity (ex: Azure Pipeline, Azure Kubernetes Service)
+	if os.Getenv("NEW_E2E_ENVIRONMENT") != "AzurePipeline" {
+		svm.InvalidateScenario()
```

Comment:
Nit: logging skips may be a good idea-- But there's no proper mechanism for this at the moment.

InvalidateScenario was intended to basically be "in no context does this scenario ever make sense, or should it ever exist", like local to local.

Reply:
I can add a svm.Skip thing and enhance the e2e framework then.